ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -1961,6 +2357,211 @@
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		A9B315591B3940610001CB9C /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
```

Comment:
Thanks, I kept 
- DYLIB_COMPATIBILITY_VERSION
- DYLIB_CURRENT_VERSION
- INFOPLIST_FILE
- LD_RUNPATH_SEARCH_PATHS

Looking at the iOS target the others do not seem necessary.